src/stdlib/Run.ts

```diff
@@ -36,16 +36,10 @@ function runFiles(interpreter: Interpreter, filenames: BrsString[], args: BrsTyp
         return BrsInvalid.Instance;
     }
 
-    try {
```

Comment:
I wouldn't mind keeping the try/catch here to maintain consistency with RBI?  Since [roca](https://github.com/hulu/roca) uses `_brs_.runInScope` leaving the default `Run` in "silent mode" should be safe while still giving us the debugging boost we need 😃

Reply:
That makes sense, I'll revert the change